apps/finance/app/src/abi/token-symbol-bytes.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+      "constant": true,
+      "inputs": [],
+      "name": "symbol",
```

Comment:
We should do token name as well in this PR, as `ds-token` also uses `bytes32` for `token.name()`